### PR TITLE
Update ThunderWaveDescription

### PR DESCRIPTION
**Pull request by rjd1922**
Change Thunder Wave description from "A move that may cause paralysis." to "A move that causes paralysis." since it always paralyzes.

### CODE REVIEW

File: data/moves/descriptions.asm [hunk 1]
```diff
@@ -478,6 +478,9 @@ WillOWispDescription:
 
 StunSporeDescription:
 ThunderWaveDescription:
+	db   "A move that"
+	next "causes paralysis.@"
+
 GlareDescription:
```
Comment (Rangi42) [inline data/moves/descriptions.asm:484]: Shouldn't Glare also be updated?

Comment (rjd1922) [reply data/moves/descriptions.asm:484]: Sorry it took me forever to see this. Yes, Glare should probably have the same description as Thunder Wave saying it always paralyzes (it it hits; Glare only has 75% accuracy.

Comment (rjd1922) [reply data/moves/descriptions.asm:484]: Actually, based on how Hypnosis's description says it "MAY cause the foe to fall asleep," while Spore says it "PUTS the foe to sleep," I assume moves with 100% accuracy are supposed to say they "may" cause a status effect.

Comment (rjd1922) [reply data/moves/descriptions.asm:484]: I made a new pull request that includes more changes.

**Comment by itsdarsh** [comment]
Closing this just because it's redundant with a larger pr.

**Pull request closed without merging**